keyboards/jopr/info.json

```
@@ -0,0 +1,562 @@
+        },
+        {
+          "label": "MX_B1",
+          "x": 1,
+          "y": 1.25
+        },
+        {
+          "label": "MX_C1",
+          "x": 2,
+          "y": 1.25
+        },
+        {
+          "label": "MX_D1",
+          "x": 3,
+          "y": 1.25
+        },
+        {
+          "label": "MX_E1",
+          "x": 4,
+          "y": 1.25
+        },
+        {
+          "label": "MX_F1",
+          "x": 5,
+          "y": 1.25
+        },
+        {
+          "label": "MX_G1",
+          "x": 6,
+          "y": 1.25
+        },
+        {
+          "label": "MX_H1",
+          "x": 7,
+          "y": 1.25
+        },
+        {
+          "label": "MX_I1",
+          "x": 8,
+          "y": 1.25
+        },
+        {
+          "label": "MX_J1",
+          "x": 9,
+          "y": 1.25
+        },
+        {
+          "label": "MX_A7",
+          "x": 10,
+          "y": 1.25
+        },
+        {
+          "label": "MX_B7",
+          "x": 11,
+          "y": 1.25
+        },
+        {
+          "label": "MX_C7",
+          "x": 12,
+          "y": 1.25
+        },
+        {
+          "label": "MX_D7",
+          "x": 13,
+          "y": 1.25,
+          "w": 2
+        },
+        {
+          "label": "MX_E7",
+          "x": 15.25,
+          "y": 1.25
+        },
+        {
+          "label": "MX_F7",
+          "x": 16.25,
+          "y": 1.25
+        },
+        {
+          "label": "MX_G7",
+          "x": 17.25,
+          "y": 1.25
+        },
+        {
+          "label": "MX_H7",
+          "x": 18.25,
+          "y": 1.25
+        },
+        {
+          "label": "MX_A2",
+          "x": 0,
+          "y": 2.25,
+          "w": 1.5
+        },
+        {
+          "label": "MX_B2",
+          "x": 1.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_C2",
+          "x": 2.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_D2",
+          "x": 3.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_E2",
+          "x": 4.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_F2",
+          "x": 5.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_G2",
+          "x": 6.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_H2",
+          "x": 7.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_I2",
+          "x": 8.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_J2",
+          "x": 9.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_A8",
+          "x": 10.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_B8",
+          "x": 11.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_C8",
+          "x": 12.5,
+          "y": 2.25
+        },
+        {
+          "label": "MX_D8",
+          "x": 13.75,
+          "y": 2.25,
+          "w": 1.25,
+          "h"; 2
```

Had a typo in my review comment, this is my fault.

```suggestion
          "h": 2
```

---

Nobody's perfect ;-)

---

:smile: